### ISSUE TITLE
Error: "acc is not a function"

### ISSUE THREAD

**Issue by bugstamp** [Urigo/graphql-modules#474]
I use graphql-modules with plain JS, apollo server and express. For subscriptions i use "subscriptions-transport-ws" package both on the client and the server. And when i'm pushing subscriptions object from app module to the SubcriptionServer, i get the "TypeError: acc is not a function at importsOnOperationCompleteHooks.reduce (@graphql-modules\core\src\graphql-module.ts:654:50)" error after each operation. But when i extract onOperation/onOperationComplete hooks from the subscriptions and pass only onConnect/onDisconnect - everything is working well.

Simple server code example:

```
const startServer = ({ schema, subscriptions }) => {
  const app = express();
  const apolloServer = new ApolloServer({
    schema,
    formatError,
    context: session => session,
  });

  app.use(express.static(paths.public));
  apolloServer.applyMiddleware({ app, path: apolloPath });

  app.get('*', (req, res) => {
    res.sendFile(paths.html);
  });

  const ws = createServer(app);
  ws.listen(port, () => {
    console.log(`Server ready at ${apolloUrl}`);
    console.log(`Subscriptions ready at ${wsUrl}`);

    SubscriptionServer.create({
      schema,
      execute,
      subscribe,
      ...subscriptions,
    }, {
      server: ws,
      path: '/subscriptions',
    });
  });
};

startServer(AppModule);
```

**Comment by ardatan** [comment]
Could you try it with the following canary version?
```
@graphql-modules/core@0.8.0-alpha.90c55362
```

**Comment by bugstamp** [comment]
Thanks for prompt reply.

Yes, it's working with canary version. No errors anymore, but onOperation/onOperationComplete hooks don't work in the provider. onInit/onRequest/onConnect/onDisconnect ones work well.
As far as i have noticed they must be available from 0.7.0 version.
Could you help me out with this?

**Comment by ardatan** [comment]
@bugstamp Could you try it with that version?
0.8.0-alpha.9f6edd2a

**Comment by bugstamp** [comment]
Yes, it works.

**Comment by ardatan** [comment]
Fixed in 0.7.6!
